tutorials/3d/procedural_geometry/introduction.rst

```diff
@@ -0,0 +1,106 @@
+.. _doc_procedural_geometry_introduction:
+
+Introduction to procedural geometry
+===================================
```

Comment:
I think this is a great topic for a tutorial. FWIW--in relation to reviewing the tutorial content--it might be helpful to know what audience you have in mind in terms of their experience (with Godot, with the concept of procedural generation, with geometry).

Based on the wording of "Geometry is a fancy way of saying shape. In computer graphics..." I'm picturing a reader who has little no experience with procedural generation nor computer-based geometry.

If my impression matches your intended target audience then it might be good to also include some examples of procedural generation (e.g. images and/or descriptions) and why someone might want to generate geometry in a procedural manner, rather than, say in 3D design software.

Reply:
Good idea. The idea was to capture a reader who has heard of procedural generation and wants to know what that means. We often get questions about whether or not Godot can "do procedural generation" because people see games like no mans sky. So this tutorial is meant to clear up what that means in a Godot-centered workflow